packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -177,7 +177,9 @@ export class VaMemorableDate {
     const target = event.target as HTMLInputElement;
 
     let [currentYear, currentMonth, currentDay] = (this.value || '').split('-');
-    if (target.classList.contains('input-month') || target.classList.contains('usa-form-group--month-input') || target.classList.contains('usa-form-group--month-select')) {
+    if (target.classList.contains('input-month') || target.classList.contains('usa-form-group--month-input')
+      || target.classList.contains('usa-form-group--month-select')
+      || target.classList.contains('usa-form-group--month-input')) {
```

Comment:
Everything looks good! 

But am I reading this correctly that `target.classList.contains('usa-form-group--month-input')` is added twice? I'm not sure why we would need to add it again to this condition.